service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java

```diff
@@ -117,4 +117,11 @@ private LinkInfo getLinkInfoFromLinkedAccount(LinkedAccount linkedAccount) {
     return ResponseEntity.ok(
         getLinkInfoFromLinkedAccount(linkedAccountWithPassportAndVisas.getLinkedAccount()));
   }
+
+  @Override
+  public ResponseEntity<Void> deleteLink(String provider) {
```

Comment:
Do you think we should create an explicit test for this controller method? There’s not too much going on in it that hasn't already been tested, but I think our approach in the past had been to test all the controller methods regardless, right?

Reply:
yes

Reply:
Okay sounds good. I actually already got started on writing a test for this on Friday, so I would be happy to wrap that up this afternoon, and maybe create a `TestUtils` class in the process. Sound reasonable?

Also, if you prefer to merge this now, go ahead and I can create a separate PR for the test.